test/common/schemaform/actions.unit.spec.js

```diff
@@ -193,4 +194,195 @@ describe('Schemaform actions:', () => {
       });
     });
   });
+  describe('uploadFile', () => {
+    let fetchMock;
+    let oldFetch;
```

Comment:
Yeah, that would probably be good, if we could figure out a reasonable way to make it reusable. There might be libraries for it, too.